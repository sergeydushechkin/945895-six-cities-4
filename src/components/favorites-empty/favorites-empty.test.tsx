import React from "react";
import renderer from "react-test-renderer";
import FavoritesEmpty from "./favorites-empty.jsx";

it(`Render FavoritesEmpty`, () => {
  const tree = renderer
    .create(
        <FavoritesEmpty />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
