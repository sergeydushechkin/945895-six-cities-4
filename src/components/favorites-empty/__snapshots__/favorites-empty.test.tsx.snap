// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render FavoritesEmpty 1`] = `
Array [
  <h1
    className="visually-hidden"
  >
    Favorites (empty)
  </h1>,
  <div
    className="favorites__status-wrapper"
  >
    <b
      className="favorites__status"
    >
      Nothing yet saved.
    </b>
    <p
      className="favorites__status-description"
    >
      Save properties to narrow down search or plan yor future trips.
    </p>
  </div>,
]
`;
